cnf-certification-test/accesscontrol/suite.go

```diff
@@ -649,3 +655,126 @@ func Test1337UIDs(env *provider.TestEnvironment) {
 
 	testhelper.AddTestResultLog("Non-compliant", badPods, tnf.ClaimFilePrintf, ginkgo.Fail)
 }
+
+type ContainerSCC struct {
+	HostDirVolumePlugin    bool
+	HostIPC                bool
+	HostNetwork            bool
+	HostPID                bool
+	HostPorts              bool
+	PrivilegeEscalation    bool
+	PrivilegedContainer    bool
+	RunAsUser              bool
+	ReadOnlyRootFilesystem bool
+	RunAsNonRoot           bool
+	Capabilities           string
+}
+
+var (
+	catagory1 = ContainerSCC{false,
+		false,
+		false,
+		false,
+		false,
+		true,
+		false,
+		true,
+		false,
+		false,
+		""}
+
+	catagory2 = ContainerSCC{false,
+		false,
+		false,
+		false,
+		false,
+		true,
+		false,
+		false,
+		false,
+		true,
+		""}
+
+	catagory3 = ContainerSCC{false,
+		false,
+		false,
+		false,
+		false,
+		true,
+		false,
+		true,
+		false,
+		false,
+		"NET_ADMIN, NET_RAW"}
+	catagory4 = ContainerSCC{false,
+		false,
+		false,
+		false,
+		false,
+		true,
+		false,
+		true,
+		false,
+		false,
+		"IPC_LOCK, NET_ADMIN, NET_RAW"}
+)
+
+func testContainerSCC(env *provider.TestEnvironment) {
```

Comment:
This test needs a `testhelper.AddTestResultLog` to fail if any `badCut`'s exist.